File: pset1/mario/less/mario.c
// Mario program - Less comfortable
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Height, h and initialise it
    int h = -1;
    while (h < 0 || h > 23)
    {
        h = get_int("Height: ");
    }

    // For loop on displaying the result
    for (int i = 1; i <= h; i++)
    {
        for (int j = h; j >= 1; j--)
        {
            if (j > i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("#\n");
    }
}

File: pset6/mario/less/mario.py
# Mario program - Less comfortable
from cs50 import get_int

while True:
    # Height, h and initialise it
    h = get_int("Height: ")
    if h >= 0 and h <= 23:
        break

# For loop on displaying the result
for i in range(1, h + 1):
    for j in range(h, 0, -1):
        print(" " if j > i else "#", end="")
    print("#")